addons/resource/models/resource_calendar.py

```diff
@@ -193,6 +222,15 @@ def _get_hours_per_day(self, attendances):
 
         return float_round(hour_count / float(number_of_days), precision_digits=2)
 
+    def _get_days_per_week(self):
+        # If the employee didn't work a full day, it is still counted, i.e. 19h / week (M/T/W(half day)) -> 3 days
+        self.ensure_one()
+        attendances = self.attendance_ids.filtered(
+            lambda attendance_id: attendance_id._is_work_period() and attendance_id.duration_hours)
+        attendances_set = attendances.mapped(lambda attendance: (attendance.week_type, attendance.dayofweek))
+        number_of_days = len(attendances_set)
+        return  number_of_days/ 2 if self.two_weeks_calendar else number_of_days
```

Comment:
```suggestion
        return  number_of_days / 2 if self.two_weeks_calendar else number_of_days
```
please, let this cute operator breathe :pleading_face:

Reply:
if two tasks are really linked; you can put one task in waiting :)